### PR TITLE
Create layout variants & Use LinearLayout

### PR DESCRIPTION
**Pull request by shashank205**
1.2.2 - Create layout variants & use LinearLayout

### CODE REVIEW

File: app/src/main/res/layout-sw600dp/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <Button
+        android:id="@+id/button_toast"
+        android:layout_width="372dp"
```
Comment (ghost) [inline app/src/main/res/layout-sw600dp/activity_main.xml:11]: Why is the width hardcoded to specific value?

File: app/src/main/res/layout-sw600dp/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        android:textColor="@android:color/white"
+        android:textSize="60sp"
+        app:layout_constraintEnd_toStartOf="@+id/button_count"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/button_count"
+        android:layout_width="372dp"
```
Comment (ghost) [inline app/src/main/res/layout-sw600dp/activity_main.xml:27]: Hardcoded width

File: app/src/main/res/layout-sw600dp/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        android:background="#FFFF00"
+        android:gravity="center_vertical"
+        android:text="@string/count_initial_value"
+        android:textAlignment="center"
+        android:textColor="@color/colorPrimary"
+        android:textSize="200sp"
```
Comment (ghost) [inline app/src/main/res/layout-sw600dp/activity_main.xml:53]: What is the difference between dp and sp?

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout-sw600dp/activity_main.xml (hunk 1 above)
Comment (shashank205) [reply app/src/main/res/layout-sw600dp/activity_main.xml:11]: "Expand Horizontally" option was used

File: app/src/main/res/layout-sw600dp/activity_main.xml (hunk 2 above)
Comment (shashank205) [reply app/src/main/res/layout-sw600dp/activity_main.xml:27]: "Expand Horizontally" option was used

File: app/src/main/res/layout-sw600dp/activity_main.xml (hunk 3 above)
Comment (shashank205) [reply app/src/main/res/layout-sw600dp/activity_main.xml:53]: sp is like dp but it is affected by user's font size preference

**Pull request closed without merging**